### DISCUSSION TITLE
flight model like a sailor plane

### DISCUSSION THREAD

**Discussion by plyky51** [flybywiresim/aircraft#3420] [Help] [answered]
Hello,
after i installed the latest version, i feel like the flight model is different and incorrect.
when you flare for landing or if you put the engines on iddle, the airplane glides a lot  (the airspeed decrease very slowly).
when i'm on the ILS, to follow the glide slope arround 140kts + full flaps , the NG is close to iddle  ( 23/24 % ) 

any idea ?

**Comment by ChruutvoLuzi** [comment, marked as answer]
Check out the Custom Fly-by-Wire Version on the installer, landing feels alot better and fixes that "issue"

**Reply by plyky51** [reply]
i tested but i didn't see a really big difference, too much inertia. even the speed break is not efficient and reactive.

**Reply by ChruutvoLuzi** [reply]
> i tested but i didn't see a really big difference, too much inertia. even the speed break is not efficient and reactive.

you definitely didnt then, the cfbw adds the flare law which commands ailerons and elevators to push the aircraft down in a sort of direct law

**Reply by wernair2k21** [reply]
Hi, 
I am experiencing the same problem as plyky51 - when pulling the throttle to IDLE on short final, the 320nx does barely bleed off the airspeed. The behavior is the same on the the Stable, the Dev and the CFWB version. When reverting back to the original MS FS2020 320neo model it works "normal".
I am happy for any suggestions or hints to try.
Thanks!

**Comment by wernair2k21** [comment]
Hi again, 
are there only few people experience this problem?
Tried again several times, with latest Custom FBW (9e6857d) and todays FS update - sailing all the way till mid of runway during all approaches.
I need to "force" the aircraft down, resulting in a hard and unstable landing.
Any ideas?
Thank you!

**Reply by hotshotp** [reply]
approach at a slower speed, it is realistic and confirmed by many pilots

**Reply by JJKHub** [reply]
World Update 3 introduced this issue. It was fixed 2 days ago in the FBW development build. Check if you still experience the problem after updating and using the dev build. For other planes asobos [workaround](https://forums.flightsimulator.com/t/flight-dynamics-bug-details/368499) might help until the sim is updated.

**Reply by wernair2k21** [reply]
I experienced the problem already before the world update 3; as said with all FBW builds. Tried against this afternoon with the FBW Custom built. Still think it is not natural, my workaround for now is pulling to idle at 500 ft AGL and disconnecting the ATHR. The speed reduction is marginal however it then leads me to a kind of normal flare...

**Reply by JJKHub** [reply]
I am currently not able to verify this, but I would say you should have approximately the following behavior (forward to minute 22):
https://youtu.be/tjtIUuCMSn8
The custom FBW build was used. The video is a month old, but the behavior including N1 power etc. should not have drastically changed.
If the behavior is different for you and Without it being a common issue for users, it will be hard to figure out what exactly goes wrong without video or a way to reproduce.
Sorry I couldn‘t help you.

**Reply by plyky51** [reply]
when you take off just after rotation, switch off the 2 engines. Me, i can glide a lot without stall, not realistic for me... (i'm ATPL)

**Reply by wernair2k21** [reply]
Thanks for your replies. I no not have any other add-ons or mods installed.
I will try some manually flown approach as shown in the video and share my experience.

**Reply by CptBayer** [reply]
I didn t get the plane into flare or better it is hard to get it to flare.. Maybee a thing which belongs to the TCA stick?! Also when using the rudder after touchdown, the aircraft lift with one wing into the air... that´s crazy ... 

The Problem during landing with flpas and thrust is another thread of mine .... 

Greets klaus

**Comment by plyky51** [comment]
anybody has the same problem ? i tested again with v.053 ...

**Reply by ChruutvoLuzi** [reply]
Use the latest dev, it has been fixed there
